pkg/cmd/cli/cmd/login.go

```diff
@@ -123,7 +124,7 @@ func NewCmdLogin(fullName string, f *osclientcmd.Factory, reader io.Reader, out
 
 	templater := templates.Templater{
 		UsageTemplate: templates.MainUsageTemplate(),
-		Exposed:       []string{"certificate-authority", "insecure-skip-tls-verify"},
+		Exposed:       []string{"certificate-authority", "insecure-skip-tls-verify", "api-version"},
```

Comment:
I don't think we need to explicitly show it in `login` help. Just let it be part of the global options help like in other commands